app/policies/transcription_policy.rb

```diff
@@ -0,0 +1,30 @@
+class TranscriptionPolicy < ApplicationPolicy
+  delegate :editor?, :approver?, :viewer?, to: :project_policy
+
+  def update?
+    admin? || (logged_in? && editor?)
+  end
+
+  def project_policy
+    ProjectPolicy.new(user, projects)
+  end
+
+  def projects
```

Comment:
I think you might want to inline this essentially private method to the `project_policy` method then pass in the `projects` to the policy for delegation

Reply:
Why? The means by which the associated project ids are gathered are different for every resource. Are you suggesting ditching the `projects` method entirely and going with

 ```
def project_policy
   ProjectPolicy.new(user, records.collect(&:workflow).collect(&:project)
end
```

for the sake of readability? Not sure what needs delegation.

Reply:
I'd say for readability in the calling method reading `project_policy` right now i have to find the `projects` method to figure out what records are passed in. I personally prefer this as i can see the state of play in one go.
```
def project_policy
  project_records = records.collect(&:workflow).collect(&:project)
  ProjectPolicy.new(user, project_records)
end
```

Re delegation, that was off topic but your `editor?` etc methods are delegating to this `project_policy` for determining transcription policy permissions.

Reply:
OK, inlined and switched to Project.where to avoid extra db calls.